src/main/java/com/google/cloud/tools/maven/AppEngineFlexibleDeployer.java

```diff
@@ -135,33 +135,40 @@ public void deployQueue() throws MojoExecutionException {
   }
 
   @VisibleForTesting
+  static final String projectError =
+      "Deployment projectId must be configured on the appengine-maven-plugin using "
```

Comment:
s/`appengine`/`app`/

Reply:
the plugin is actually appengine-maven-plugin, this repo is just called app-maven-plugin because the appengine-maven-plugin repo points to the appcfg based implementation.

Reply:
Ah right